### PR TITLE
Toolbar integration

### PR DESCRIPTION
**Pull request by annda**
Adds simple buttons for inserting `<doi>` and `<isbn>` syntax.

The icons in the DOI dropdown are just favicons from the respective ID resolver websites:

image

I'm not sure if the icons are recognizable for the users? Any suggestions?

### CODE REVIEW

**Comment by splitbrain** [comment]
DOIs and ISBNs are easily distinguishable from each other (one having a slash, the other consisting of `[0-9Xx]{10|13}`. So I don't think needing two separate buttons is needed. Instead the an approach similar to the vshare plugin could ask for the identifier and then insert the correct syntax.

(ISBNs should be accepted with dashes in them)

File: script.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    /**
+     * Ask for ID and check format to determine DOI or ISBN syntax
+     */
+    buildSyntax: function () {
+
+        const ident = prompt(LANG.plugins.doi.prompt);
+        if (!ident) return;
+
+        const isbnRegex = new RegExp('^(?=(?:\\D*\\d){10}(?:(?:\\D*\\d){3})?$)[\\d-]+$', 'i');
```
Comment (splitbrain) [inline script.js]: This seems not to match any ISBNs ending in X

https://regex101.com/r/NOZNF9/2

File: img/doi.png
Comment (splitbrain) [inline img/doi.png:1]: no longer needed?

File: img/ISBN.png
Comment (splitbrain) [inline img/ISBN.png:1]: no longer needed?

File: style.less [hunk 2]
```diff
@@ -11,3 +11,7 @@
         font-size: 0.8em;
     }
 }
+
+div.picker.plugin_doi_picker_narrow {
+    width: auto;
+}
```
Comment (splitbrain) [inline style.less]: I think this is obsolete as well?

**Review by splitbrain — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (annda) [reply script.js]: Fixed: https://regex101.com/r/NOZNF9/3

**Pull request merged**